### PR TITLE
Adjust Video Preview component to new designs

### LINKED ISSUE #982: Update video tile design

### PR DESCRIPTION
**Pull request by rafalpawlow**
Close #982

### CODE REVIEW

File: src/shared/components/VideoPreviewBase/VideoPreviewBase.styles.tsx [hunk 1]
```diff
@@ -57,7 +64,11 @@ export const CoverContainer = styled.div<ClickableProps>`
   transition: all ${transitions.timings.regular} ${transitions.easing};
   cursor: ${(props) => (props.clickable ? 'pointer' : 'auto')};
 
-  :hover {
+  :active {
+    ${() => clickableAnimation(false)}
```
Comment (drillprop) [inline src/shared/components/VideoPreviewBase/VideoPreviewBase.styles.tsx:68]:
```suggestion
    ${clickableAnimation(false)}
```

File: src/shared/components/VideoPreviewBase/VideoPreviewBase.styles.tsx [hunk 2]
```diff
@@ -208,22 +219,49 @@ export const CoverVideoPublishingStateOverlay = styled.div`
...
 export const PublishingStateText = styled(Text)`
   margin-left: ${sizes(1.5)};
 `
 
+export const KebabMenuIconContainer = styled.div<{ isActive?: boolean }>`
+  display: ${({ isActive }) => (isActive ? 'flex' : 'none')};
+  align-items: center;
+  justify-content: center;
+  width: 36px;
+  height: 36px;
```
Comment (drillprop) [inline src/shared/components/VideoPreviewBase/VideoPreviewBase.styles.tsx]: let's maybe use `sizes` here
```suggestion
  width: ${sizes(9}};
  height: ${sizes(9}};
```

File: src/shared/components/VideoPreviewBase/VideoPreviewBase.styles.tsx [hunk 3]
```diff
@@ -208,22 +219,49 @@ export const CoverVideoPublishingStateOverlay = styled.div`
...
+  path {
+    transition: all ${transitions.timings.regular} ${transitions.easing};
+  }
+
+  &:hover {
+    path:not([fill='none']) {
+      fill: ${colors.white};
+    }
+
+    background-color: ${transparentize(1 - 0.06, colors.white)};
```
Comment (drillprop) [inline src/shared/components/VideoPreviewBase/VideoPreviewBase.styles.tsx]: I think we could use here `colors.transparentWhite[6]`. Also probably not your choice, but this particular color used in this context is barely visible. Do we plan changing background around video tiles in the future? Personally I don't like it,  maybe we could change it to more brighter variant?

**Review by drillprop — COMMENTED** [review]
Generally looks good, found some small things

File: src/shared/components/VideoPreviewBase/VideoPreviewBase.styles.tsx (hunk 1 above)
Comment (drillprop) [reply src/shared/components/VideoPreviewBase/VideoPreviewBase.styles.tsx:68]: My bad, I think I know why you did that. 
Hmm, let's just ignore this then

File: src/shared/components/VideoPreviewBase/VideoPreviewBase.styles.tsx (hunk 3 above)
Comment (rafalpawlow) [reply src/shared/components/VideoPreviewBase/VideoPreviewBase.styles.tsx]: Good catch. After investigation, it turned out, there was a change in background color.

File: src/shared/theme/sizes.ts [hunk 4]
```diff
@@ -8,11 +10,18 @@ export function sizes(n: number, raw?: boolean) {
...
   header: 100,
   sheetOverlay: 150,
   nearSheetOverlay: 160,
   sideNav: 200,
-  overlay: 10,
-  nearOverlay: 20,
   globalOverlay: 999,
 }
+
+export function square(size: string | number): SerializedStyles {
```
Comment (drillprop) [inline src/shared/theme/sizes.ts]: nice utility function!
```suggestion
export function square(size: string | number) {
```
but we don't need to specify return type here.

**Review by drillprop — APPROVED** [review]

**Pull request merged**